doc/onboarding.md

```diff
@@ -32,33 +33,38 @@ onboarding session.
 ## Local setup
 
   * git:
-    * make sure you have whitespace=fix: `git config --global --add apply.whitespace fix`
-    * usually PR from your own github fork
+    * Make sure you have whitespace=fix: `git config --global --add apply.whitespace fix`
+    * Always continue to PR from your own github fork
+      * Branches in the nodejs/node repository are only for release lines
     * [See "Updating Node.js from Upstream"](./onboarding-extras.md#updating-nodejs-from-upstream)
-    * make new branches for all commits you make!
+    * Make new branches for all commits you make!
 
-  * notifications:
-    * use [https://github.com/notifications](https://github.com/notifications) or set up email
-    * watching the main repo will flood your inbox, so be prepared
+  * Notifications:
+    * Use [https://github.com/notifications](https://github.com/notifications) or set up email
+    * Watching the main repo will flood your inbox (several hundred notifications on typical weekdays), so be prepared
 
   * `#node-dev` on [webchat.freenode.net](https://webchat.freenode.net/) is the best place to interact with the CTC / other collaborators
+    * If there are any questions after the session, this is a good place to ask!
+    * Presence is not mandatory, but please drop a note there if force-pushing to `master`
 
 
 ## Project goals & values
 
-  * collaborators are effectively part owners
-    * the project has the goals of its contributors
+  * Collaborators are effectively the collective owners of the project
+    * The project has the goals of its contributors
 
-  * but, there are some higher-level goals and values
-    * not everything belongs in core (if it can be done reasonably in userland, let it stay in userland)
-    * empathy towards users matters (this is in part why we onboard people)
-    * generally: try to be nice to people
+  * There are some higher-level goals and values
+    * Not everything belongs in core (if it can be done reasonably in userland, let it stay in userland)
+    * Empathy towards users matters (this is in part why we onboard people)
```

Comment:
@refack I think "being kind and courteous" is too low a bar. I agree that "empathy" can be hard to define specifically and may not be understood by everyone, but I prefer it to "be nice" which (to me, at least) minimizes things.

Maybe what's needed here is a link on the word "empathy" to another doc (or another part of this doc) that might explain things more fully. Probably out-of-scope for this PR, though. And I'm certainly open to other ideas.